composer.json

```
@@ -15,7 +15,7 @@
         "jean85/pretty-package-versions": "~1.5.0",
         "mglaman/phpstan-drupal": "^0.12.4",
```

See https://github.com/mglaman/drupal-check/issues/202#issuecomment-773722443
```suggestion
        "mglaman/phpstan-drupal": "^0.12.8",
```